packages/p2p/src/components/app.jsx

```diff
@@ -157,7 +157,9 @@ const App = observer(props => {
                             <Tabs
                                 onTabItemClick={general_store.handleTabClick}
                                 active_index={general_store.active_index}
-                                className='p2p-cashier'
+                                className={classNames('p2p-cashier', {
+                                    'p2p-cashier__advertiser-tab': is_mobile && general_store.is_advertiser,
```

Comment:
Is `general_store.is_advertiser` necessary? I think when `should_show_verification` is false, it means that the user is an advertiser and hence `general_store.is_advertiser` would be true. Can you try with just `isMobile` and see if that's enough?

Reply:
yes we need it because we should check if we have 3 or 4 tabs